tools/accuracy_checker/accuracy_checker/adapters/classification.py

```diff
@@ -37,13 +37,17 @@ def parameters(cls):
             'argmax_output': BoolField(
                 optional=True, default=False, description="identifier that model output is ArgMax layer"
             ),
+            'block': BoolField(
```

Comment:
please document all added parameters in readme

Reply:
Fixed